Formula/homebrew-dns-scout.rb

```diff
@@ -0,0 +1,19 @@
+class HomebrewDnsScout < Formula
```

Comment:
```suggestion
class DnsScout < Formula
```
Just call this `dns-scout`. You'll need to change the filename too.